### PR TITLE
JAMES-3768 Allow disabling Cassandra Mail queue view

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by Arsnael** [comment]
```
Unable to create injector, see the following errors:
1) [Guice/BindingAlreadySet]: CassandraMailQueueViewConfiguration was bound multiple times.
Bound at:
1  : TestRabbitMQModule.configure(TestRabbitMQModule.java:58)
      \_ installed by: Modules$OverrideModule -> Modules$OverrideModule -> Modules$OverrideModule -> TestRabbitMQModule
2  : CassandraMailQueueViewModule.getMailQueueViewConfiguration(CassandraMailQueueViewModule.java:51)
      \_ installed by: Modules$OverrideModule -> Modules$OverrideModule -> Modules$OverrideModule -> Modules$OverrideModule -> Modules$CombinedModule -> Modules$CombinedModule -> Modules$CombinedModule -> Modules$CombinedModule -> Modules$CombinedModule -> CassandraMailQueueViewModule
```

File: server/apps/distributed-app/docs/modules/ROOT/pages/configure/rabbitmq.adoc [hunk 1]
```diff
@@ -113,6 +113,11 @@ this implementation relies on Cassandra.
 |===
 | Property name | explanation
 
+|cassandra.view.enabled
```
Comment (Arsnael) [inline server/apps/distributed-app/docs/modules/ROOT/pages/configure/rabbitmq.adoc:116]:
```suggestion
| cassandra.view.enabled
```

**Review by quantranhong1999 — COMMENTED** [review]
Read it

**Review by Arsnael — APPROVED** [review]

File: server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/api/FakeMailQueueView.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import org.apache.commons.lang3.NotImplementedException;
+import org.apache.james.queue.api.ManageableMailQueue;
+import org.apache.james.queue.rabbitmq.EnqueueId;
+import org.apache.james.queue.rabbitmq.EnqueuedItem;
+import org.apache.james.queue.rabbitmq.MailQueueName;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class FakeMailQueueView<V extends ManageableMailQueue.MailQueueItemView> implements MailQueueView<V> {
```
Comment (vttranlina) [inline server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/api/FakeMailQueueView.java:33]: `NoneMailQueueView` is easier to imagine with me

**Review by vttranlina — APPROVED** [review]

**Review by quantranhong1999 — APPROVED** [review]

**Comment by quantranhong1999** [comment]
```
07:26:09,542 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.0:check (check-style) on project james-server-guice-cassandra: You have 1 Checkstyle violation. -> [Help 1]
```

**Review by Arsnael — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

**Pull request merged**